helm/stunner-gateway-operator/templates/stunner-default-dataplane.yaml

```diff
@@ -32,11 +32,14 @@ spec:
   {{- with .affinity }}
   affinity: {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .containerSecurityContext }}
+  containerSecurityContext: {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
Unfortunately, we do not support such a field in the Dataplane object. Please take a look at the Dataplane API spec: https://pkg.go.dev/github.com/l7mp/stunner-gateway-operator/api/v1alpha1#DataplaneSpec

Reply:
hi, thanks for the response. [v1 currently has the field](https://pkg.go.dev/github.com/l7mp/stunner-gateway-operator/api/v1#DataplaneSpec). are you saying the field is deprecated? if so, do you have any recommendations for configuring the context for the Dataplane's container?

Reply:
ohh, dumb me.... I was looking at the v1alpha1. In this case, we are good with this